resources/views/tabbar/navigation/contents/it/event/connpass_list.blade.php

```diff
@@ -1,12 +1,9 @@
-<form action="" method="get">
-    <input type="hidden" name="contents" value="connpass">
-    <input type="text" class="text-input" placeholder="@{{$ctrl.yearParam}}" name="year" style="width: 50px;">年
-    <input type="text" class="text-input" placeholder="@{{$ctrl.monthParam}}" name="month" style="width: 50px;">月
-    <input type="text" class="search-input" placeholder="keyword (例)swift" name="keyword" style="width: 200px;">
-    <input type="submit" class="button button--light" value="検索">
-</form>
-<li class="list__header">
-    @{{$ctrl.getJson['results_start']}}〜@{{$ctrl.getJson['results_returned']}}件 (該当件数：@{{$ctrl.getJson['results_available']}}件)
+<span ng-repeat="count in $ctrl.getCountArray($ctrl.getJson['results_available'])" style="display:inline;">
```

Comment:
__[code_sniffer]__
* No PHP code was found in this file and short open tags are not allowed by this install of PHP. This file may be using short open tags but PHP does not allow them.